src/ekat/kokkos/ekat_kokkos_types.hpp

```diff
@@ -23,6 +24,9 @@ using DefaultDevice = Kokkos::Device<Kokkos::DefaultExecutionSpace, Kokkos::Defa
 // A device type to force host execution
 using HostDevice = Kokkos::Device<Kokkos::DefaultHostExecutionSpace, Kokkos::DefaultHostExecutionSpace::memory_space>;
 
+template<typename DT, typename... Props>
+using ViewLR = Kokkos::View<DT,Kokkos::LayoutRight,Props...>;
```

Comment:
Do we ever use LayoutLeft views? If not, this symbol is probably descriptive enough.

Reply:
@jeff-cohere , yes. I've used them in places where fortran compatibility is needed.